x-pack/plugins/observability/public/pages/overview/index.tsx

```diff
@@ -33,13 +33,9 @@ interface Props {
   routeParams: RouteParams<'/overview'>;
 }
 
-function calculatetBucketSize({ startTime, endTime }: { startTime?: string; endTime?: string }) {
-  if (startTime && endTime) {
-    return getBucketSize({
-      start: moment.utc(startTime).valueOf(),
-      end: moment.utc(endTime).valueOf(),
-      minInterval: '60s',
-    });
+function calculatetBucketSize({ start, end }: { start?: number; end?: number }) {
+  if (start && end) {
+    return getBucketSize({ start, end, minInterval: '60s' });
   }
 }
```

Comment:
```suggestion
function calculatetBucketSize({ start, end }: { start: number; end: number }) {
    return getBucketSize({ start, end, minInterval: '60s' });
}
```

This is a good example how one optional variable causes a cascade of optionals everywhere. It's always better to make an argument required if possible.

Reply:
Explanation why https://github.com/elastic/kibana/pull/71259#discussion_r452744232